executor/insert_common.go

```
@@ -291,6 +293,17 @@ func (e *InsertValues) handleErr(col *table.Column, val *types.Datum, rowIdx int
 		err = types.ErrWarnDataOutOfRange.GenWithStackByArgs(colName, rowIdx+1)
 	} else if types.ErrTruncated.Equal(err) {
 		err = types.ErrTruncated.GenWithStackByArgs(colName, rowIdx+1)
+	} else if types.ErrTruncatedWrongVal.Equal(err) && (colTp == mysql.TypeDuration || colTp == mysql.TypeDatetime) {
+		valStr, err1 := val.ToString()
+		if err1 != nil {
+			logutil.BgLogger().Warn("truncate value failed", zap.Error(err1))
```

We'd better ignore the error here instead of log it.

---

好的👌

---

The log has been removed.